docs/storage/storage.md

```diff
@@ -98,3 +98,29 @@ comment the next property:
   <description>notebook persistence layer implementation</description>
 </property>
 ```   
+
+</br>
+#### Notebook Storage in ZeppelinHub  <a name="ZeppelinHub"></a>
+
+ZeppelinHub storage layer allows out of the box connection of Zeppelin instance with your ZeppelinHub account. First of all, you need to either comment out the following  property in **zeppelin-site.xml**:
```

Comment:
Currently this documentation is shown like the below:
![screen shot 2016-05-20 at 11 18 04 am](https://cloud.githubusercontent.com/assets/10060731/15415330/f3aeeedc-1e7c-11e6-8868-6923aa6a10e8.png)

So you have to add a new line between the each code blocks and sentences. Then it will show like:

![screen shot 2016-05-20 at 11 20 00 am](https://cloud.githubusercontent.com/assets/10060731/15415360/1d385a86-1e7d-11e6-9601-f410149b56fc.png)